web/src/pages/billing/SequencingGroupsByMonth.tsx

```diff
@@ -101,21 +150,100 @@ const SequencingGroupsByMonthDisplay: React.FunctionComponent<ISGByMonthDisplayP
     const containerRef = useRef<HTMLDivElement>(null)
     const [measureRef, { width, height }] = useMeasure<HTMLDivElement>()
 
+    // Merges counts, grouped by either type or technology.
+    const mergeTypeOrTechCount = React.useCallback(
```

Comment:
Given that this doesn't have any dependencies, it would be good to move it out of the component and call it as a normal function.